modules/client-configuration/pages/clients-opensuse.adoc

```diff
@@ -0,0 +1,133 @@
+[[clients-opensuse]]
+= Registering openSUSE Clients
+
+This section contains information about registering Salt clients running {ubuntu} operating systems.
+
+{productname} supports opensuse Leap 15 and 15.1 Clients using Salt.
+// Traditional clients are not supported.
+
+// [IMPORTANT]
+// ====
+// {suse} does not endorse or support {productname}.
+// ====
+
+Bootstrapping is supported for starting openSUSE clients and performing initial state runs such as setting repositories and performing profile updates.
+
+
+
+== Prepare to Register
+
+Some preparation is required before you can register {ubuntu} clients to the {productname} Server.
+
+////
+ifeval::[{suma-content} == true]
```

Comment:
I thin kwe can remove this. You already have how to add the client tools below, right?